unity-client/Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/LoadableShape.cs

```diff
@@ -25,6 +25,8 @@ public new class Model : BaseShape.Model
 
         public static LoadWrapper GetLoaderForEntity(DecentralandEntity entity)
         {
+            if(entity.meshRootGameObject == null) return null;
```

Comment:
Can we add a `Log.Warning` here? I'm concerned that we might be hiding a bigger error by adding this check here, so it'd be better if we can still see when it happens at least in debug mode or editor.

Reply:
Sure, just added it.